lib/util/js.js

```diff
@@ -80,5 +80,16 @@ module.exports = {
       isFinite(value) &&
       Math.floor(value) === value &&
       value >= 0;
+  },
+  /**
+   * Checks that a value is an integer.
+   *
+   * @param {*} value
+   * @return {Boolean}
+   */
+  isInteger: function isInteger(value) {
+    return typeof value === 'number' &&
+      isFinite(value) &&
+      Math.floor(value) === value;
```

Comment:
Looks like the tests failed with Node.js v0.10, as `isInteger` is missing: https://travis-ci.org/bitpay/bitcore/jobs/75524680#L3598

Switched it back to use the util